packages/vuelidate/test/unit/validators.fixture.js

```diff
@@ -16,3 +17,4 @@ export const isOdd = (v) => v % 2 === 1
 
 export const asyncIsEven = withAsync(toAsync(isEven, 5))
 export const asyncIsOdd = withAsync(toAsync(isOdd, 5))
+export const sameAs = (equalTo) => (value) => unwrap(value) === unwrap(equalTo)
```

Comment:
Why not use the original `sameAs` validator?

Reply:
I tried to import it from the relative path but it didn't work because it was out of scope of the tsc I think. What is the proper way to import the actual `sameAs` validator?

Reply:
Just import from `@vuelidate/validators` package, should work.

Reply:
Ah makes sense :joy: should've guessed!

Reply:
Should be good now